python/setup.py

```diff
@@ -146,6 +146,11 @@ def find_version(*filepath):
     "six >= 1.0.0",
     "flatbuffers",
     "faulthandler;python_version<'3.3'",
+    # simplejson and psutil are required by the Reporter.
+    "simplejson",
+    "psutil",
+    # aiohttp is required by the Dashboard.
+    "aiohttp",
```

Comment:
I think we can use normal json and HTTPServer.

For psutil, it would be nice to add the dep, but afaik @robertnishihara has reported it breaks for a lot of users.

Reply:
Breaks in what sense?

simplejson is only used to serialize psutil's namedtuples. A custom json encoder could be used instead if necessary (it surprises me that the standard json module doesn't have namedtuple_as_object).

Reply:
Ah, I saw this later down.
I can either re-implement this with a bunch of bash scripts or something, or just disable the dashboard if psutil is not installed. Prefer latter.

Reply:
Commit below disables the dashboard if `psutil` not present (the dashboard runs but the reporters don't send it anything, so it's just empty).